django_admin_shell/templates/django_admin_shell/shell.html

```diff
@@ -76,7 +102,8 @@ <h5> Django version : {{django_version}} </h5>
         <div class="col-xs-6 text-info">
             <span class="glyphicon glyphicon-info-sign"></span> Ctrl + Enter : Run code<br/>
             <span class="glyphicon glyphicon-info-sign"></span> Ctrl + Up/Down : Load code from history<br/>
-            <span class="glyphicon glyphicon-info-sign"></span> Tab : Indent = 4 spaces
+            <span class="glyphicon glyphicon-info-sign"></span> Tab : Indent = 4 spaces<br/>
+            <span class="glyphicon glyphicon-info-sign"></span> Double click : Save code snippet
```

Comment:
It is not clear where exactly user should double click (when I tested it took me some time when i understanded that I should dclick on code in console. Can you extend or reformat?  Maybe a tooltip could be a good option. 
On hover we can display more rich explanation.